### PR TITLE
fix: autocomplete to alpha.44

### PR DESCRIPTION
**Pull request by sbellone**
### Changes

- Updated to alpha.44
- Re-imported updated `autocomplete-theme-classic` and updated our overriding CSS:
  - Some CSS classes have been renamed (`Touch` -> `Detached`), updated or removed (`.aa-Panel--desktop`)
- Fixed detached mode CSS (background color was not respecting the theme)
- Exposed `detachedMediaQuery` to let user control the detached mode (https://algolia-autocomplete.netlify.app/docs/detached-mode/#what-is-detached)

### How to test

- yarn dev
- Reduce your windows size to trigger detached mode

### CODE REVIEW

**Comment by bodinsamuel** [comment]
This is a duplicate of #220 
Also I'm awaiting this fix https://github.com/algolia/autocomplete/commit/5991e77539f4cf2c7ab0a92c1ce626f3176348e7 to be released

**Comment by sbellone** [comment]
I hadn't seen it, but:
- We don't want to "revert" the touch UI, it's better to keep it and expose a parameter
- `_highlightResult` is not part of our record shema
- I've taken the time to update the whole CSS and make the detached UI work correctly

Let's release this one to give a workaround to people running the UI on Gatsby?

File: frontend/src/autocomplete-theme-classic.scss [hunk 1]
```diff
@@ -18,8 +18,14 @@
   --aa-text-color: rgb(38, 38, 39);
```
Comment (bodinsamuel) [inline frontend/src/autocomplete-theme-classic.scss:18]: You can replace this big c/c by importing the module directly like I did

File: public/index.html [hunk 2]
```diff
@@ -178,6 +178,7 @@ <h2>Test content</h2>
             text: '#d6d6e7',
             colorSourceIcon: '#d6d6e7'
           },
+          detachedMediaQuery: '',
```
Comment (bodinsamuel) [inline public/index.html:181]: Can we make that clearer by using true or not empty string?
or 
```json
{ "detached": true | false, "mediaQuery": string }
```

**Review by bodinsamuel — COMMENTED** [review]
- The clear button has not the correct color when hovered
- The detached input has not the same size

File: public/index.html (hunk 2 above)
Comment (sbellone) [reply public/index.html:181]: detachedMediaQuery: true sounds not very clear neither.
`{ "detached": true | false, "mediaQuery": string }` would need a parent property
Or `detached: true | { mediaQuery: string }`
We can discuss it tomorrow.

**Comment by sbellone** [comment]
> The clear button has not the correct color when hovered

What is the correct color? There is not hover on a touch device anyway.

> The detached input has not the same size

Yes it's a design choice: https://github.com/algolia/autocomplete/blob/next/packages/autocomplete-theme-classic/src/theme.scss#L482

File: public/index.html (hunk 2 above)
Comment (bodinsamuel) [reply public/index.html:181]: `detached: true | { mediaQuery: string }` seems good to me

**Comment by bodinsamuel** [comment]
> What is the correct color? There is not hover on a touch device anyway.

Well the bug is still there anyway, correct color could be white idk 

> Yes it's a design choice, to look more like a button I imagine:

it's up to us to decide our own design, and I think it's better if both type of input are the same height no?

**Comment by sbellone** [comment]
> Well the bug is still there anyway, correct color could be white idk

Ok I thought it was the Cancel button, this bug was already here and I hadn't retested this part.

For the input height, we've already shipped it like that last time and I think it makes sense to have it look like a button since it opens a fullscreen search. One less CSS property to override also.

**Review by bodinsamuel — APPROVED** [review]

**Pull request merged**